### ISSUE TITLE
Options in dropdown container should be populated on page load. 

### ISSUE THREAD

**Issue by daviddarx** [jackocnr/intl-tel-input#1391]
### Steps to reproduce
1.  Instanciate intlTelInput with a custom dropdown container. 
2.  Inspect `iti__selected-flag` and its attribute `aria-activedescendant`. 
3.  Inspect the custom dropdown container (i.e `.phone-dropdown-container`, on the screenshot). 

### Expected behaviour
According accessibility tests we made with https://www.levelaccess.com, the options in the dropdown container should be populated on page load. Currently, the options only appear in the container when the user click on the select. This lead to an error on page load  (_ARIA attribute doesn't contain a valid value_) as the referenced ID in `aria-activedescendant` isn't present on the page load. 

The ideal solution would be to populate all the options on page load. 
Alternatively, the removal of the attribute `aria-activedescendant` in the cases where the dropdown container is empty will also avoid the error to be fired.

### Actual behaviour
The dropdown container is empty on page load, leading to the error mentioned above. 

### Initialisation options
```
phoneField(input, {
    dialCodeLabel: 'Choose your country dial-in code',
    initialCountry: 'CH',
    separateDialCode: true,
    utilsScript: 'getNumber', 
    dropdownContainer: document.querySelector('#phone-dropdown-container');
  });
```
  
![screenshot_empty_dropdown_container](https://user-images.githubusercontent.com/10694809/212638721-dc8a69e7-8482-4145-8cc0-abcbcc6bf292.jpg)

**Comment by jackocnr** [comment]
Thanks for raising this. I've just pushed a change so now we only populate activedescendent when the dropdown is open. Let me know if you have any more issues.
